eng/scripts/Sdk-Generate.ps1

```diff
@@ -0,0 +1,173 @@
+param(
+$GenerateInput, 
+$GenerateOutput,
+$RepoRoot = "${PSScriptRoot}/../.."
+)
+
+$input = Get-Content $GenerateInput | ConvertFrom-Json
+$inputFiles = $input.changedFiles;
+$inputFiles += $input.relatedReadmeMdFiles;
+$inputFiles = $inputFiles | select -Unique
+Write-Output "List Of changed swagger files and related readmes" $inputFiles "`n"
```

Comment:
Use Write-Host here. Write-Output while may dump to the host in some cases it also writes to the object pipeline which we don't want for these debugging messages.